docs/usage.md

```diff
@@ -5,26 +5,42 @@
 `run.py` is the main script for ocs-ci. You can view the full usage details by passing in the `--help` argument.
 
 ```bash
-python run.py --help
+python3 run.py --help
 ```
 
 ## Required Arguments
 
 There are a few arguments that are required ocs test execution:
 
-* `--cred <cred_file>`
+* `--cred <cred_file>` - if you have aws configured by `aws configure`, see:
+    [AWS doc](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+    this parameter is not needed
 * `--suite <suite_file>`
 
 ## Useful Arguments
 
 Some non-required arguments that we end up using a lot:
 
 * `--log-level <level>` - set the log level that is output to stdout.
+* `--cluster-name <name>` - name of cluster.
+* `--cluster-path <path>` - path where to create the directory which will
+    contains all the installation/authentication information about cluster.
+    `Use this parameter when running on already deployed cluster!` You can
+    pass the cluster path from previous execution if was created automatically.
 
 ## Examples
 
-Run OCS install suite:
+* Run OCS install suite:
 
 ```bash
-python run.py --cred ~/aws.yml --suite suites/ocs_basic_install.yml --log-level info
-```
\ No newline at end of file
+python3 run.py --cred ~/aws.yml --suite suites/ocs_basic_install.yml \
```

Comment:
Same here (and code block below) and the one below, we should really encourage using a virtualenv instead of the system installed python3.

Reply:
I am using venv, but wasn't sure about how it's handel python2 or python3 . Thanks for pointing it out.. Now I really see that:

```bash
(.venv) [jenkins@qe-ocs4-slave-prod-3 ocs-ci]$ which python
~/ocs-ci/.venv/bin/python
(.venv) [jenkins@qe-ocs4-slave-prod-3 ocs-ci]$ python --version
Python 3.6.6
```